StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentMethodType.swift

```diff
@@ -151,6 +151,15 @@ extension PaymentSheet {
                 print("[Stripe SDK]: PaymentSheet could not offer these external payment methods: \(missingExternalPaymentMethods). See https://stripe.com/docs/payments/external-payment-methods#available-external-payment-methods")
             }
 
+            if
+                intent.supportsLink,
+                !recommendedStripePaymentMethodTypes.contains(.USBankAccount),
+                !intent.isDeferredIntent,
+                intent.linkFundingSources?.contains(.bankAccount) == true
```

Comment:
for `.bankAccount` to appear in `linkFundingSources` has a bunch of extra controls so this if-statement is unlikely to be "accidentally" executed

for example, there's a [special flag](https://livegrep.corp.stripe.com/view/stripe-internal/pay-server/lib/consumer/risk/command/get_allowed_link_funding_sources.rb#L296-297) that needs to be enabled

Reply:
I'm worried this might cause older SDK versions to see a half-finished version of Instant Debits when this rolls out more widely. We'll want to make sure to check server-side that the request has the minimum SDK version (the one you end up GA-ing in) *or* a flag before returning `.bankAccount`.

Reply:
Yes great point! We will make sure to account for this (this is quite common in Financial Connections too - and we even have some custom backend code/"library" to handle this type of versioning easier across platforms). It's a little sad our flag system does not seem to support versioning as well.